### ISSUE TITLE
Forgotten Mods: Detect open affixes and allow user to put possible master craft mods

### ISSUE THREAD

**Issue by thirdy** [exiletrade/exiletrade#56]
This task is done in 3 steps:
1. Produce a "blank" mod indicating an open affix. Limit this to magic or rare items. If item has unresolved mods (those hybrid mods are not yet handled), skip.

(step 1 alone is a good feature already)
1. Make map of all possible master mods for a given item. With that data, we can have a dropdown select where user can see possible master mods.
2. (Optional) Go for the extra mile, and via angular, dynamically re-calculate total and pseudo mods that are displayed.

**Comment by bolte-17** [comment]
You'd have trouble actually doing any search functionality with this, though? It'd be much stronger if implemented by the exiletools tracker.

**Comment by thirdy** [comment]
Yep, it won't be possible to search for "high res amulet with an open prefix".  I agree that having this 'forgotten mods' data in the indexer would be really great but that would require contributing perl code into exiletools-indexer; either that or this.

Since this one is probably easier to materialize, I thought why not.
